viewer/core/src/public/migration/Cognite3DViewer.ts

```diff
@@ -457,6 +595,54 @@ export class Cognite3DViewer {
     }
   }
 
+  /**
+   * Sets camera controls mode anything allowed in CameraControlsOptions type.
+   * @param controlsOptions JSON object with camera controls options.
+   */
+  setCameraControlsMode(controlsOptions: CameraControlsOptions) {
+    this._cameraControlsOptions.useOnClickTargetChange =
+      controlsOptions?.onClickTargetChange ?? this._cameraControlsOptions.useOnClickTargetChange;
+    this._cameraControlsOptions.canInterruptAnimations =
+      controlsOptions?.canInterruptAnimations ?? this._cameraControlsOptions.canInterruptAnimations;
+
+    switch (controlsOptions?.zoomToCursor) {
+      case 'disable':
```

Comment:
```suggestion
      case 'disable':
      case undefined:
```
When undefined, you want to have it disabled - currently no change happens if this is not defined (which I think is not what we want).